skt/reporter.py

```diff
@@ -286,13 +286,15 @@ def update_mergedata(self):
         else:
             mergedata = self.stateconfigdata(mergedata)
 
-        if self.cfg.get("cfgurl"):
-            response = requests.get(self.cfg.get("cfgurl"))
-            if response:
-                mergedata['config'] = response.text
-        else:
-            with open("%s/.config" % self.cfg.get("workdir"), "r") as fileh:
-                mergedata['config'] = fileh.read()
+        if not self.cfg.get('mergelog'):
```

Comment:
It will probably be OK for now, I think. Later we'll need to be able to run report after any stage/command. Including just after merge, without build.

Reply:
What if we're using `make rh-configs`? Those only appear during the `build` phase, not during `merge`. Will that cause problems?

Reply:
Yeah, no .config appears during merge. We'll not be able to run report if merge succeeded, but we didn't run build, but it's probably OK, as we don't need that right now.

Reply:
Agreed that running reports at any stage can be useful (albeit I don't think it's that useful to not build the kernel and only apply the patches). If you want, I can change the patch to work around that.